way_to_home/telegram_bot/bothelper.py

```diff
@@ -0,0 +1,68 @@
+"""This module provides helping functions for telegram bot."""
+
+import pickle
+import os
+import sys
+import django
+
+# pylint: disable=wrong-import-position
+
+SOURCE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, SOURCE_PATH)
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "way_to_home.settings")
+django.setup()
+
+from django.conf import settings
+from telebot import TeleBot, apihelper
+from utils.redishelper import REDIS_HELPER
+
+UNAUTHORIZED_ACCESS_MESSAGE = 'Для активації сповіщень через телеграм' \
+                              ' необхідно скористатись посиланням на нашому сайті.'
+TELEGRAM_REDIS_KEY = 'telegram'
+TELEGRAM_CACHE_TIME = 60*60*24
+TOKEN = settings.TELEGRAM_BOT_TOKEN
+BOT = TeleBot(token=TOKEN)
+
+
+def send_telegram_message(text, user):
+    """This function sends telegram message with given text to user"""
+    chat_id = user.user_profile.telegram_id
+    if not chat_id:
+        return False
```

Comment:
```suggestion
        def send_telegram_message(text, user):
        """This function sends telegram message with given text to user"""
            chat_id = user.user_profile.telegram_id
            if not chat_id:
                return False
```
In our case, you need here just logic of sending message.
So change param `user` to `chat_id`. Logic of retrieve `chat_id` must be in celery task.

Reply:
right, done